corehq/apps/hqwebapp/static/hqwebapp/js/requirejs_config.js

```diff
@@ -27,4 +27,12 @@ requirejs.config({
             "datatables.net": "datatables",
         },
     },
+
+    // This is really build config, but it's easier to define a js function here than in requirejs.yaml
+    // The purpose of this is to replace hqDefine and hqRequire calls, which in a requirejs context are
+    // just pass throughs to define and require, with actual calls to define and require. This is needed
+    // because r.js's dependency tracing depends on parsing define and require calls.
+    onBuildRead: function (moduleName, path, contents) {
+        return contents.replace(/hqDefine/g, 'define').replace(/hqRequire/g, 'require');
```

Comment:
having either of those function names as a substring in a longer name seems unlikely, but it would be marginally safer to use word delimiters: `/\bhqDefine\b/g` and `/\bhqRequire\b/g`